telethon/client/chats.py

```diff
@@ -1313,4 +1372,130 @@ async def get_stats(
+        reset_invite_hash: bool = None,
+        join_muted: bool = None
+    ):
+        """
+        Stuff to do with Group Calls, Possible for Administrator and
+        Creator.
+        
+        will raise ValueError, if `method` is None or Invalid.
+
+        Arguments
+            method (`str`):
+                Any of `create`, `discard`, `start_schedule` and 
+                `edit_title` will work. Check Examples to know more Clearly.
+
+            entity (`int` | `str`):
+                Username/ID of Channel or Chat, where to Modify Group Call.
+
+            schedule (`hints.Datelike`, `optional`):
+                Used to Schedule the GroupCall.
+
+            title (`str`, `optional`):
+                Edits the Group call Title. It should be used with
+                `edit_title` as method parameter.
+
+            reset_invite_hash (`bool`, `optional`):
+                Reset the Invite Hash of Group Call of Specific Channel or 
+                Chat.
+
+            join_muted (`bool`, `optional`):
+                Whether the New Group Call Participant should be Muted or
+                Not.
+
+        Method Parameter :
+            - `create` : `Create a Group Call or Schedule It.`
+            - `discard` : `Stop a Group Call.`
+            - `edit_title` : `Edit title of Group Call.`
+            - `edit_perms` : `Edit Permissions of Group Call.`
+            - `start_schedule` : `Start a Group Call which was Scheduled.`
+
+        Returns
+            The resulting :tl:`Updates` object.
+
+        Raises
+            ChatAdminRequiredError - You Should be Admin, in order to
+            Modify Group Call.
+
+        Example
+        .. code-block:: python
+
+                # Starting a Group Call
+                await client.modify_groupcall("create", -100123456789)
+
+                # Scheduling a Group Call, within 5-Minutes
+                from datetime import timedelta
+                schedule_for = timedelta(minutes=5)
+                await client.modify_groupcall(
+                    "create",
+                    schedule=schedule_for
+                )
+
+                # Editing a Group Call Title
+                new_title = "Having Fun with Telethon"
+                await client.modify_groupcall(
+                    "edit_title",
+                    title=new_title
+                )
+
+                # Stopping/Closing a Group Call
+                await client.modify_groupcall("discard", -100123456789)
+
+                # Force Start Scheduled Group Call
+                await client.modify_groupcall("start_schedule", chat)
+
+                # Toggle Group Call Setting
+                await client.modify_groupcall(
+                    "edit_perms",
+                    reset_invite_hash=True,
+                    join_muted=True) # Mute New Group call Participants
+        """
+        if not method:
+            raise ValueError("Method Cant be None.")
+
+        if method == "create":
```

Comment:
`modify_groupcall` works very unlike any other method in the library, which I don't think is a good fit. The raw API methods used here seem to also be very simple -- perhaps how to do this stuff should instead live in the wiki?